src/Symfony/Component/String/Tests/AbstractUnicodeTestCase.php

```diff
@@ -86,6 +86,10 @@ public function testCodePointsAt(array $expected, string $string, int $offset, i
             $this->markTestSkipped('Skipping due to issue ICU-21661.');
         }
 
+        if (3 !== grapheme_strlen('☢☎❄') && '☢☎❄' === $string) {
+            $this->markTestSkipped('Skipping due to issue in PCRE2 (https://github.com/PCRE2Project/pcre2/issues/361).');
```

Comment:
What about doing this in the data provider instead (i.e. not yield the fixture when the length is incorrectly detected)?

Reply:
Why not, should this be done also for the check above this one?

Reply:
Done 🙂